zScanner/ViewControllers/Documents/DocumentsListViewController.swift

```diff
@@ -83,12 +83,15 @@ class DocumentsListViewController: BaseViewController, ErrorHandling {
                     self.rightBarButtons = []
```

Comment:
Please remove the `rightBarButtons` if they are not used anymore in the view controller